vscode/src/player/Player.ts

```diff
@@ -52,12 +52,22 @@ export default class Player {
       this.process = new Promise((resolve) => (this.closeCodioResolver = resolve));
       await this.codeEditorPlayer.moveToFrame(0);
       this.play(this.codeEditorPlayer.events, this.relativeActiveTime);
-      commands.executeCommand('setContext', 'inCodioSession', true);
+      this.updateContext('inCodioSession', true);
     } catch (e) {
       console.log('startCodio failed', e);
     }
   }
 
+  /**
+   * Update given context to given value and update manager.
+   * @param context String representing context to update.
+   * @param value Value to set given context string to.
+   */
+  private updateContext(context: string, value: any): void {
+    commands.executeCommand('setContext', context, value);
+    FSManager.update();
```

Comment:
why is this needed?

Reply:
To refresh icons when the view is reevaluated.

Reply:
This causes the order of the codios to change which is kind of weird. Looks like this happens also when you add a new Codio and the view is refreshing but it is much more visible when it happens on every play. I wonder how we could solve that

Reply:
Let's sort the tree items by their label to ensure the same order each time an update is called.

Reply:
Added Codio and Metadata interfaces and sorted found codios by their name property.